### PR TITLE
3923 email template

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: shared/src/business/utilities/emailGenerator/emailTemplates/PetitionService.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      <div>
+        <div>For IRS only:</div>
+        <div>docketNumber: {caseDetail.docketNumber}</div>
+        <div>docketEntryNo: {docketEntryNumber}</div>
+        <div>documentId: {documentDetail.documentId}</div>
+        <div>eventCode: {documentDetail.eventCode}</div>
+      </div>
+
+      <p>
+        <i>
```
Comment (kkoskelin) [inline shared/src/business/utilities/emailGenerator/emailTemplates/PetitionService.jsx]: nit: we prefer to use `<em>` over `<i>` for accessibility of semantic html.

**Review by kkoskelin — APPROVED** [review]
Nice.

File: shared/src/business/useCaseHelper/service/sendIrsSuperuserPetitionEmail.js [hunk 2]
```diff
@@ -1,33 +1,69 @@
...
     applicationContext,
     defaultTemplateData: {
-      caseCaption: 'undefined',
-      docketNumber: 'undefined',
-      documentName: 'undefined',
-      loginUrl: 'undefined',
-      name: 'undefined',
-      serviceDate: 'undefined',
-      serviceTime: 'undefined',
+      emailContent: '',
```
Comment (codyseibert) [inline shared/src/business/useCaseHelper/service/sendIrsSuperuserPetitionEmail.js]: any clue if empty string is allowed?  I find it strange we used 'undefined' previous for everything.

Comment (matthopson) [reply shared/src/business/useCaseHelper/service/sendIrsSuperuserPetitionEmail.js]: Yeah, it's very strange. I'll look into this more.

Comment (matthopson) [reply shared/src/business/useCaseHelper/service/sendIrsSuperuserPetitionEmail.js]: From AWS docs:
>DefaultTemplateData — (String)
A list of replacement values to apply to the template when replacement data is not specified in a Destination object. These values act as a default or fallback option when no other data is available.
>
>The template data is a JSON object, typically consisting of key-value pairs in which the keys correspond to replacement tags in the email template.

So I think the intention is to be able to do something like:

```
DefaultTemplateData: {
  name: 'Human Person',
}

ReplacementTemplateData: {
  name: 'Joe Exotic',
}
```
```
// Email Template

Dear {{name}},

I hope the food is good there.

Regards,
Carol Baskin
```

File: web-api/serverless-cases.yml [hunk 3]
```diff
@@ -111,6 +111,7 @@ provider:
     NODE_ENV: ${self:custom.vars.nodeEnv}
     EMAIL_SOURCE: noreply@mail.efcms-${opt:stage}.${opt:domain}
     EMAIL_SERVED_TEMPLATE: case_served_${opt:stage}
```
Comment (rachaelparris) [inline web-api/serverless-cases.yml:113]: Sidenote - I think we should rename this `case_served` template as part of DOD for this story. It's really for a document served.

Comment (matthopson) [reply web-api/serverless-cases.yml:113]: I think @codyseibert is refactoring this :)

**Pull request merged**